backend/app/services/subscription/unified_executor.py

```
@@ -259,6 +276,138 @@ async def _execute_sse_sync(
+        task_id=execution_data.task_id,
+        subtask_id=execution_data.subtask_id,
+    )
+
+    try:
+        # Dispatch task to device via WebSocket
+        dispatch_task = asyncio.create_task(
+            execution_dispatcher.dispatch(request, device_id=device_id, emitter=emitter)
+        )
+
+        # Wait for completion and collect results
+        accumulated_content, final_event = await emitter.collect()
```

Ruff (0.15.4)

[warning] Unpacked variable `final_event` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)